src/main.rs

```diff
@@ -265,8 +267,14 @@ fn safe_main(program: &str, args: &[String]) -> Fallible<()> {
         return Err(UsageError { message: "invalid number of arguments".to_string() }.into());
     };
 
+    let profiler = match matches.opt_str("cpu_profile") {
+        Some(path) =>
+            Some(sandboxfs::ScopedProfiler::start(&path).context("Failed to start CPU profile")?),
+        None => None,
+    };
     sandboxfs::mount(mount_point, &options, &mappings, ttl, input, output)
         .context(format!("Failed to mount {}", mount_point.display()))?;
+    drop(profiler);
```

Comment:
Why?

Reply:
Just to make it explicit that we are interested in profiling only the `mount()` loop. I can take it out though; don't feel strongly about it.

Reply:
Oh, but turns out I have to remove it in order for your comment above to work:

```
error[E0381]: use of possibly uninitialized variable: `profiler`                                                                                                                              
   --> src/main.rs:276:10                                                                                                                                                                     
    |                                                                                                                                                                                         
276 |     drop(profiler);                                                                                                                                                                     
    |          ^^^^^^^^ use of possibly uninitialized `profiler`                                                                                                                              
                                                                                                                                                                                              
error: aborting due to previous error                                                                                                                                                         
```

So removed.